.github/workflows/docker.yml

```diff
@@ -0,0 +1,33 @@
+name: Build Docker Image
+
+on: 
+  push:
+    branches:
+      - main
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+          password: ${{ secrets.DOCKERHUB_USERNAME }}
+      -
+        name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: superserious/bot:latest
```

Comment:
SHouldn't this be a branch?

Reply:
Nope, that's the target of the Docker image that we generate. We always want the newest image to be at `superserious/bot:latest`.